### ISSUE TITLE
qcint 4.1.1 fails to build

### ISSUE THREAD

**Issue by susilehtola** [sunqm/qcint#8]
Updating the Fedora packages fails in
```
/builddir/build/BUILD/qcint-4.1.1/src/g2e_coulerf.c: In function 'CINTg0_2e_coulerf':
/builddir/build/BUILD/qcint-4.1.1/src/g2e_coulerf.c:236:31: error: incompatible type for argument 2 of 'CINTrys_roots'
  236 |         CINTrys_roots(nroots, x, u, w, count);
      |                               ^
      |                               |
      |                               double *
In file included from /builddir/build/BUILD/qcint-4.1.1/src/g2e_coulerf.c:31:
/builddir/build/BUILD/qcint-4.1.1/src/rys_roots.h:24:39: note: expected 'double' but argument is of type 'double *'
   24 | void CINTrys_roots(int nroots, double x, double *u, double *w);
      |                                ~~~~~~~^
/builddir/build/BUILD/qcint-4.1.1/src/g2e_coulerf.c:236:9: error: too many arguments to function 'CINTrys_roots'
  236 |         CINTrys_roots(nroots, x, u, w, count);
      |         ^~~~~~~~~~~~~
In file included from /builddir/build/BUILD/qcint-4.1.1/src/g2e_coulerf.c:31:
/builddir/build/BUILD/qcint-4.1.1/src/rys_roots.h:24:6: note: declared here
   24 | void CINTrys_roots(int nroots, double x, double *u, double *w);
      |      ^~~~~~~~~~~~~
/builddir/build/BUILD/qcint-4.1.1/src/g2e_coulerf.c: In function 'CINTg0_2e_coulerf_simd1':
/builddir/build/BUILD/qcint-4.1.1/src/g2e_coulerf.c:364:31: error: incompatible type for argument 2 of 'CINTrys_roots'
  364 |         CINTrys_roots(nroots, x, u, w, 1);
      |                               ^
      |                               |
      |                               double *
In file included from /builddir/build/BUILD/qcint-4.1.1/src/g2e_coulerf.c:31:
/builddir/build/BUILD/qcint-4.1.1/src/rys_roots.h:24:39: note: expected 'double' but argument is of type 'double *'
   24 | void CINTrys_roots(int nroots, double x, double *u, double *w);
      |                                ~~~~~~~^
/builddir/build/BUILD/qcint-4.1.1/src/g2e_coulerf.c:364:9: error: too many arguments to function 'CINTrys_roots'
  364 |         CINTrys_roots(nroots, x, u, w, 1);
      |         ^~~~~~~~~~~~~
In file included from /builddir/build/BUILD/qcint-4.1.1/src/g2e_coulerf.c:31:
/builddir/build/BUILD/qcint-4.1.1/src/rys_roots.h:24:6: note: declared here
   24 | void CINTrys_roots(int nroots, double x, double *u, double *w);
      |      ^~~~~~~~~~~~~
make[2]: *** [CMakeFiles/cint.dir/build.make:563: CMakeFiles/cint.dir/src/g2e_coulerf.c.o] Error 1
make[2]: Leaving directory '/builddir/build/BUILD/qcint-4.1.1/x86_64-redhat-linux-gnu'
make[2]: *** Waiting for unfinished jobs....
make[2]: Entering directory '/builddir/build/BUILD/qcint-4.1.1/x86_64-redhat-linux-gnu'
```

**Comment by sunqm** [comment]
Fixed in v4.1.2

**Comment by susilehtola** [comment]
Thanks, 4.1.2 compiles successfully.
